src/controller/python/chip-device-ctrl.py

```diff
@@ -639,6 +694,23 @@ def start_rpc_server():
             print("\nKeyboard interrupt received, exiting.")
             sys.exit(0)
 
+def __get_response_dict(status: StatusCodeEnum, result: Optional[Dict[Any, Any]] = None, error:Optional[str] = None) -> Dict [Any, Any]:
+    if error is not None:
+        return { "status" : status.value, "error" :f'exception {error}' }
+    else:
+        if result is not None:
+            return { "status" : status.value, "result": result}
+        else:
+            return { "status" : status.value}
+
+def __check_supported_os()-> bool:
+    if platform.system() == 'Darwin':
+        raise Exception(platform.system() + " not supported")
+    elif sys.platform.startswith('linux'):
```

Comment:
Should we worry about upper-case, lower-case issues here?

Reply:
Resolved.